dallinger/default_configs/global_config_defaults.txt

```diff
@@ -45,4 +45,6 @@ num_dynos_worker = 1
 prolific_api_token = Set your Prolific API token in ~/.dallingerconfig!
 prolific_api_version = v1
 prolific_estimated_completion_minutes = 0
+prolific_project = Project
```

Comment:
I think that 'Dallinger experiments' might be a better default for `prolific_project`.